vortex-array/src/builders/varbinview.rs

```diff
@@ -120,6 +140,11 @@ impl VarBinViewBuilder {
     // buffers adjusted.
     // The views must all point to sections of the buffers and the validity length must match
     // the view length.
+    //
```

Comment:
Can you make this a full doc instead of having a comment preceding a doc string?